process-engine/process_model.use_cases
# ✨Add super admin claim

## Changes

1. Add a claim check for the `can_manage_process_instances` claim, which indicates a super admin.
    - These admins can always delete ProcessModels, regardless of their other claims.
2. Replace TSLint with ESLint
3. Replace Gulp with a purely TSC based build
4. Apply new ESLint styleguide
5. Allow linter errors to cause Jenkins build failures

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/336

PR: #4

## How to test the changes

Best use the [Runtime](https://github.com/process-engine/process_engine_runtime/pull/338) and our [IdentityServer](https://github.com/process-engine/identity_server)

- Deploy a ProcessModel with any user
- Use `bob` to delete the ProcessModel => Has neither the SuperAdmin, nor the CanDeleteProcessModel claim, so it should not work
- Use `alice` to delete the ProcessModel => Has the SuperAdmin claim, so it should work
